editoast/src/views/timetable/stdcm.rs

```diff
@@ -100,6 +99,10 @@ enum StdcmError {
     TrainSimulationFail,
     #[error("Path items are invalid")]
     InvalidPathItems { items: Vec<InvalidPathItem> },
+    #[error("Invalid consist mass: {message}")]
+    InvalidConsistMass { message: String },
+    #[error("Invalid consist length: {message}")]
+    InvalidConsistLength { message: String },
```

Comment:
It would have been great to have numerical values instead of a generic message string :/

Because of that, 1) the full error message is split in several places and 2) in tests you have to `assert_eq` on strings instead of numerical values which are more stable.